src/contribute/design-documents.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: section
+title: 'Beam Design Documents'
+section_menu: section-menu/contribute.html
+permalink: /contribute/design-documents/
+---
+
+# Design Documents
+
```

Comment:
I think it will be useful to say a sentence like "this is a collection of documents that may or may not be up to date"

Reply:
Done